### PR TITLE
JI:765258 P2022-1250 2. Add redux logger middleware

### PR DESCRIPTION
**Pull request by pawelwojdas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,14 +72,16 @@
     "core-js": "^3.6.5",
     "fuse.js": "^6.5.3",
     "html-loader": "^3.1.0",
-    "i18next": "^21.6.7",
     "i18next-browser-languagedetector": "^6.1.2",
+    "i18next": "^21.6.7",
```
Comment (kacperChlon) [inline package.json:76]: Please move @types uuid to devDeps

File: package.json [hunk 2]
```diff
@@ -72,14 +72,16 @@
...
     "lodash-es": "^4.17.21",
-    "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-i18next": "^11.15.3",
     "react-is": "17.0.2",
+    "react-redux": "^7.2.6",
     "react-router-dom": "^6.2.1",
+    "react": "17.0.2",
+    "redux-logger": "^3.0.6",
```
Comment (kacperChlon) [inline package.json:84]: please add https://www.npmjs.com/package/@types/redux-logger as well and add configuration to the logger with options you find useful or nice to have :)

File: package.json (hunk 1 above)
Comment (pawelwojdas) [reply package.json:75]: Moved

File: package.json (hunk 2 above)
Comment (pawelwojdas) [reply package.json:83]: `@types/redux-logger` is added already in devDeps. In the last commit I added some configuration to the logger.

**Review by kacperChlon — APPROVED** [review]

**Pull request merged**